ci/travis/move_bazel_rule.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Probably shouldn't check this in, it would be sufficient to leave it in the github PR description for future reference.

Reply:
Fair enough. Pushing a change removing it